frontend/src/context/SessionContext.tsx

```diff
@@ -46,6 +57,24 @@ export const SessionProvider: FC = ({ children }) => {
     [history, login, setCookie, user]
   );
 
+  const handleRegister = useCallback(
+    async (username, password, displayName) => {
+      const result = await register({ variables: { username, password, displayName } });
+
+      if (result.data.register.token) {
+        setCookie('token', result.data.register.token, { maxAge: 86400 });
+        setUser(result?.data?.register?.user);
+
+        await createCart({
+          variables: { userId: result?.data?.register?.user?._id, status: EnumCartStatus.WAITING },
+        });
```

Comment:
Can create cart after register in backend ?

Reply:
@Icyscools Totally agree.